docker-compose.yaml

```
@@ -0,0 +1,67 @@
+version: '3'
+services:
+  app:
+    build:
+      context: .
+      dockerfile: docker/src/Dockerfile
+    ports:
+      - "8000:8000"
+    depends_on:
+      - postgres
+      - minio
+    env_file:
+      - .env
+    environment:
+      DATABASE_URL: postgresql://postgres:postgres@postgres:5432/homework_db
+      MINIO_ROOT_USER: minio_user
+      MINIO_ROOT_PASSWORD: minio_password
+      MINIO_HOST: minio:9000
+    restart: always
+    networks:
+      - backend-network
+
+
+  postgres:
+    image: postgres
+    ports:
+      - "5440:5432"
```

Думаю, для разработки можно и не менять дефолтный порт с `5432` на `5440`
```suggestion
      - "5432:5432"
```